src/plugins/condition/components/Condition.vue

```diff
@@ -68,13 +68,15 @@
                                         <input v-model="domainObject.configuration.name"
                                                class="t-condition-input__name"
                                                type="text"
+                                               :disabled="!telemetry.length"
                                         >
                                     </span>
                                 </li>
                                 <li>
                                     <label>Output</label>
                                     <span class="controls">
                                         <select v-model="selectedOutputKey"
+                                                :disabled="!telemetry.length"
```

Comment:
Hmm. Wondering if it matters that the user could change the condition name or the output value if there is no telemetry dragged into the conditionSet.
This would also prevent the default condition's name and output from being changed when the default condition is not dependent on any telemetry being present.

Reply:
Asking Charles. Was following pattern of SW where widget can't be edited until telemetry is added.

Reply:
The current SW pattern sucks - you should be able to edit the default style without telemetry.

Reply:
next question then; should user be allowed to add conditions to the condition set with no telemetry added (beyond the default condition)?

Reply:
I made the changes in the trigger any/all PR (https://github.com/nasa/openmct/pull/2688)